### PR TITLE
Console: UI for environment flows studio

### PR DESCRIPTION
**Pull request by ThibaudAV**
## Issue

https://gravitee.atlassian.net/browse/APIM-5875

## Description

UI for environment flows studio
- edit dialog
- studio for policies

image
image
image

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-btrblrmqat.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/settings/environment-flows/environment-flows-studio/environment-flows-studio.component.ts [hunk 1]
```diff
@@ -14,17 +14,77 @@
...
+
+  public onDelete(): void {
+    // TODO
+  }
+
+  public onDeploy(): void {
+    // TODO
+  }
+
+  public onSave(): void {}
```
Comment (ytvnr) [inline gravitee-apim-console-webui/src/management/settings/environment-flows/environment-flows-studio/environment-flows-studio.component.ts]: not a `// TODO` here ? :p

File: gravitee-apim-console-webui/src/shared/components/specific-json-schema-type/resource-type.component.html [hunk 2]
```diff
@@ -35,6 +35,10 @@
   <mat-hint *ngIf="props.description || props.hintStart as hint">
     {{ hint }}
   </mat-hint>
+
+  <mat-hint *ngIf="resourceNotExist$ | async">
+    Resource doesn't exist. Please take care to add it with the API so that it is taken into account.
```
Comment (ytvnr) [inline gravitee-apim-console-webui/src/shared/components/specific-json-schema-type/resource-type.component.html]: `does not`
@jourdiw should we prefer "long" form ?

**Review by ytvnr — APPROVED** [review]

File: gravitee-apim-console-webui/src/shared/components/specific-json-schema-type/resource-type.component.html (hunk 2 above)
Comment (jourdiw) [reply gravitee-apim-console-webui/src/shared/components/specific-json-schema-type/resource-type.component.html]:
```suggestion
    Resource does not exist. Please add it to the API for it to be taken into account.
```

Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/shared/components/specific-json-schema-type/resource-type.component.html]: ~~censored joke~~

🚀

**Pull request merged**